src/net/server/world/World.java

```diff
@@ -782,8 +782,8 @@ public void runMountSchedule() {
     public void resetBossEntries(){
         try{
             Connection con = DatabaseConnection.getConnection();
-            PreparedStatement ps = con.prepareStatement("UPDATE boss_entries SET papEntries = ?, zakumEntries = ?, horntailEntries = ?, pinkbeanEntries = ?");
-            for(int i = 1; i < 5; i++){
+            PreparedStatement ps = con.prepareStatement("UPDATE boss_entries SET papEntries = ?, zakumEntries = ?, horntailEntries = ?, pinkbeanEntries = ?, fantasybossEntries = ?");
```

Comment:
Now that I think about it, why can't we just put in 0's in the statement and skip inserting them afterwards?

Reply:
Probably could do that as well haha, but leaves for flexibility to change later.